cromwell.examples.conf

```diff
@@ -573,22 +573,36 @@ backend {
 services {
   MetadataService {
     config {
-      # Set this value to "Inf" to turn off metadata summary refresh.  The default value is currently "2 seconds".
-      # metadata-summary-refresh-interval = "Inf"
-      # For higher scale environments, e.g. many workflows and/or jobs, DB write performance for metadata events
-      # can improved by writing to the database in batches. Increasing this value can dramatically improve overall
-      # performance but will both lead to a higher memory usage as well as increase the risk that metadata events
-      # might not have been persisted in the event of a Cromwell crash.
-      #
-      # For normal usage the default value of 200 should be fine but for larger/production environments we recommend a
-      # value of at least 500. There'll be no one size fits all number here so we recommend benchmarking performance and
-      # tuning the value to match your environment.
-      # db-batch-size = 200
-      #
-      # Periodically the stored metadata events will be forcibly written to the DB regardless of if the batch size
-      # has been reached. This is to prevent situations where events wind up never being written to an incomplete batch
-      # with no new events being generated. The default value is currently 5 seconds
-      # db-flush-rate = 5 seconds
+      # For the standard MetadataService implementation, cromwell.services.metadata.impl.MetadataServiceActor:
+        # Set this value to "Inf" to turn off metadata summary refresh.  The default value is currently "2 seconds".
```

Comment:
```hocon
# Don't indent  
    # like this.
```

Line up the hash marks, or IntelliJ will do it for the next editor who uses the pretty-formatter.

```hocon
# Can still indent
#     like this.
```